src/app/styles.tsx

```diff
@@ -0,0 +1,139 @@
+import styled from 'styled-components';
+import COLORS from '@/styles/colors';
+import { openSans } from '@/styles/fonts';
+import image from '../../public/HomePage.webp';
+
+export const PageContainer = styled.div`
+  width: 100%;
+  min-height: 100vh;
+`;
+
+export const TitleSection = styled.div`
+  width: 100%;
+  height: 100vh; // Use 100vh to make the div fill the entire viewport height
+  background: linear-gradient(
+      222deg,
+      rgba(40, 120, 194, 0.5) 14.2%,
+      rgba(0, 50, 96, 0.5) 76.27%
+    ),
+    url(${image.src}) no-repeat center / cover;
+  display: flex;
+  flex-direction: column;
+  padding-top: 200px;
+  padding-right: 440px;
+  padding-left: 84px;
+  gap: 30px;
+`;
+
+export const ButtonDiv = styled.div`
+  width: 244px;
+  height: 40px;
+`;
+
+export const StatisticsSection = styled.div`
+  width: 100%;
+  height: 146;
+  background-color: white;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  box-shadow: 0px 8px 20px rgba(0, 0, 0, 0.2);
```

Comment:
`box-shadow` seems to not be working since `StatisticsSection` and the `HowYouCanHelpContainer` are sibling elements. Could we try to find a fix in this PR?